src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldSymbol.cs

```diff
@@ -221,7 +222,14 @@ public sealed override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
 
         public sealed override string GetDocumentationCommentXml(CultureInfo preferredCulture = null, bool expandIncludes = false, CancellationToken cancellationToken = default(CancellationToken))
         {
```

Comment:
>{ [](start = 8, length = 1)

nit: could be factored `return SourceDocumentationCommentUtils.GetAndCacheDocumentationComment(this, expandIncludes, ref expandIncludes ? ref _lazyExpandedDocComment : ref _lazyDocComment);` or the `ref` could be extracted as a local.
This applies to other files too.

Reply:
➡️ Updated all 6 cases in C# code